### PR TITLE
Add bg language

### PR DESCRIPTION
**Pull request by nikolalardev**

### CODE REVIEW

**Review by peter279k — APPROVED** [review]

File: src/resources/lang/bg/read-time.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'reads_left_to_right' => true,
+    'min' => 'мин.',
```
Comment (mtownsend5512) [inline src/resources/lang/bg/read-time.php]: Could you please remove the ``.`` character so it is more consistent with other translations?

File: src/resources/lang/bg/read-time.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    'minute' => 'минута',
+    'sec' => 'сек.',
```
Comment (mtownsend5512) [inline src/resources/lang/bg/read-time.php]: Could you please remove the ``.`` character so it is more consistent with other translations?

**Review by mtownsend5512 — CHANGES_REQUESTED** [review]
Could you please remove the ``.`` character from the ``min`` and ``sec`` keys so it is more consistent with other translations?

**Comment by mtownsend5512** [comment]
Thanks for your contribution!

**Pull request merged**